bot/bot.py

```diff
@@ -66,6 +70,21 @@ def embed_template(self, **kwargs):
         kwargs['color'] = self.color
         return discord.Embed(**kwargs)
 
+    @commands.Cog.listener()
+    async def on_ready(self):
+        """ Synchronize the guilds the bot is in with the guilds table. """
+        await self.db_helper.sync_guilds(*self.guilds)
+
+    @commands.Cog.listener()
+    async def on_guild_join(self, guild):
+        """ Insert the newly added guild to the guilds table. """
+        await self.db_helper.insert_guilds(guild)
+
+    @commands.Cog.listener()
+    async def on_guild_remove(self, guild):
+        """ Delete the recently removed guild from the guilds table. """
+        await self.db_helper.delete_guilds(guild)
```

Comment:
This function should probably be `delete_guild` singular

Reply:
The way the function works is you can pass it as many guild arguments you want and it will delete all of them. Look up the unpacking operator `*`. Figured I didn't need one function for removing one at a time and another for multiple.

Reply:
Yeah I spotted this when I got further down, it's quite useful actually!